hw/top_earlgrey/data/ip/chip_adc_ctrl_testplan.hjson

```diff
@@ -3,71 +3,94 @@
 // SPDX-License-Identifier: Apache-2.0
 {
   name: chip_adc_ctrl
-  testpoints: [
-    // ADC_CTRL (pre-verified IP) integration tests:
```

Comment:
Might we want to be able to have comments?

Reply:
Loosing comments seems like a big problem, do we have many of them? Are there no parsers that can save the comments?

Reply:
I still need to find a solution for the comments.